pkg/graph/schema.graphql

```diff
@@ -49,6 +49,7 @@ ModelPlan represent the data point for plans about a model. It is the central da
 type ModelPlan {
   id: UUID!
   modelName: String!
+  abbreviation: String

```

Comment:
Same here - will want to add `abbreviation` to [ModelPlanChanges](https://github.com/CMSgov/mint-app/pull/590/files#diff-7b3b137cd7a3e86ef5717c83df62cd6be4aa416eb0d4d7eb1ebc7ecba3bf92c4L131) so that the frontend can properly update these fields.

Reply:
EASI-2899 also calls out updating unit tests - adding these new fields into our existing tests would probably be all we need!

Reply:
I don't see any tests regarding plan general characteristics, but I have updated the model plan test.